common/views/themes/mixins.ts

```diff
@@ -6,31 +6,14 @@ export function gridCells(columns: number, key: ColumnKey): string {
       const col = c + 1;
       return `
       .grid__cell--${key}${col} {
-        flex-basis: ${(col / columns) * 100}%;
-        max-width: ${(col / columns) * 100}%;
-      }
-
-      .grid__cell--shift-${key}${col} {
-        margin-left: ${(col / columns) * 100}%;
-      }
-    `;
-    })
-    .join(' ');
-}
+        grid-column: span ${col};
 
-export function cssGridCells(columns: number, key: ColumnKey): string {
-  return [...Array(columns).keys()]
-    .map(c => {
-      const col = c + 1;
-
-      return `
-      .css-grid__cell--${key}${col} {
-        flex-basis: ${(col / columns) * 100}%;
-        max-width: ${(col / columns) * 100}%;
+        &.grid__cell--shift-${key}1 {
```

Comment:
I don't think this is the right approach in the long term, but for now it lets us keep the shape of the objects we use to create grids that we are currently using.

I think maybe if we'd started with css grid, we'd want to use objects that describe a starting track of the grid, and a number of columns to span (rather than a number of columns to span and a possible 'shift')

so e.g. where we currently have `{m: 8, shiftM: 2}` we could have something like `{m: [3, 8]}`

Reply:
(I made the Styled component)